RIOT-OS/RIOT
# boards/common/blxxxpill: Fix mixup in pinout

### Contribution description

TX0 and RX0 in the pinout got mixed up. This swaps them back.

### Testing procedure

Check https://github.com/RIOT-OS/RIOT/blob/f22ce155bba88ea37d0fa73660d8ca99a6830fb2/boards/common/blxxxpill/include/periph_conf.h#L191-L203 and confirm that the RX and TX assignment to pins matches the labeling to pins in the SVG and CSV file.

### Issues/PRs references

None
